src/lib/util/bids/bids.ts

```diff
@@ -13,22 +13,30 @@ export const sortBidsByTime = (bids: Bid[]) => {
 
 /**
  * Sorts a list of bids, by amount, from highest to lowest.
+ * Returns sorted bids by amount and highest bid.
  * @param bids list of bids to sort
  */
 export const sortBidsByAmount = (bids: Bid[]) => {
-	return bids?.sort((a, b) =>
+	const sortedBids = bids?.sort((a, b) =>
```

Comment:
Also mentioned in another PR - I have refactored this file and `Actions`. I pulled over the updated utils to this PR to avoid conflicts/errors